app/Http/Controllers/StatusController.php

```diff
@@ -126,6 +126,9 @@ public static function getDashboard($user): Paginator {
                      ->select('statuses.*')
                      ->orderBy('train_checkins.departure', 'desc')
                      ->whereIn('user_id', $followingIDs)
+                     ->where('visibility', 0)
+                     ->orWhere('visibility', 2)
```

Comment:
Maybe save one line of code?
```suggestion
                     ->whereIn('visibility', [0, 2])
```